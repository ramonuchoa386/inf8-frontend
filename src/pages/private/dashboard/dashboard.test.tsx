import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { rendererCreate } from '../../../utils/test/test-utils';

import Page from '.';

describe('Dashboard', () => {
  test('Snapshot', () => {
    const page = rendererCreate(
      <MemoryRouter>
        <Page />
      </MemoryRouter>
    ).toJSON();
    expect(page).toMatchSnapshot();
  });
});
