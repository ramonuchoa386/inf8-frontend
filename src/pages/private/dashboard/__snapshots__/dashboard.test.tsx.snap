// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Snapshot 1`] = `
.c4 {
  border: 1px solid #FF805E;
  color: #FF805E;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  background-color: transparent;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: bold;
  padding: 3px 9px;
  border-radius: 100px;
  display: inline-block;
  font-size: 14px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4:hover {
  -webkit-transition: all 0.7s ease;
  transition: all 0.7s ease;
  color: #FFF;
  background-color: #FF805E;
}

.c4:not(:last-of-type) {
  margin-right: 12px;
}

.c2 {
  width: 100%;
  background-color: #CFCFC4;
  padding: 17px 20px;
  -webkit-transition: all 0.4s ease-in-out;
  transition: all 0.4s ease-in-out;
  height: 54px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  overflow-y: auto;
}

.c5 {
  width: 100%;
  padding: 50px 20px 65px;
  height: 100vh;
}

.c5 code {
  color: #FF805E;
}

.c5 h3 {
  margin-bottom: 10px;
}

.c5 h5,
.c5 h6 {
  margin: 20px 0px 10px;
}

.c1 {
  width: 100%;
}

@media only screen and (min-width:768px) {
  .c2 {
    padding: 17px 60px;
  }
}

@media only screen and (min-width:1366px) {
  .c2 {
    padding: 17px 75px;
  }
}

@media only screen and (min-width:1024px) {
  .c0 {
    height: 100vh;
  }
}

@media only screen and (min-width:768px) {
  .c5 {
    padding: 50px 60px 65px;
  }
}

@media only screen and (min-width:1366px) {
  .c5 {
    padding: 50px 75px 65px;
  }
}

@media only screen and (min-width:768px) {
  .c1 {
    overflow-y: auto;
  }
}

<div
  className="c0"
>
  <div />
  <div
    className="c1"
  >
    <div
      className="c2"
      data-testid="HeaderID"
    >
      <div
        className="c3"
      >
        <span>
          Est√° Logado? 
          false
        </span>
        <div
          className=""
        >
          <button
            className="c4"
            onClick={[Function]}
          >
            Entrar
          </button>
        </div>
      </div>
    </div>
    <div
      className="c5"
    >
      <div>
        Dashboard
        <br />
        <a
          href="/apis"
          onClick={[Function]}
        >
          apis
        </a>
        <br />
        <a
          href="/cwp"
          onClick={[Function]}
        >
          cwp
        </a>
        <br />
        <a
          href="/otk"
          onClick={[Function]}
        >
          otk
        </a>
        <br />
        <a
          href="/logs"
          onClick={[Function]}
        >
          logs
        </a>
        <p>
          What is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. Why do we use it? It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like). Where does it come from? Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32. The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham. Where can I get some? There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary, making this the first true generator on the Internet. It uses a dictionary of over 200 Latin words, combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable. The generated Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc.
        </p>
      </div>
    </div>
  </div>
</div>
`;
